Thanks for looking this up.
Would you mind adjusting the `test_darwin_installer()` test to match, the error was:
```
>           raise AssertionError(_error_message()) from cause
E           AssertionError: expected call not found.
E           Expected: check_output(['brew', 'install', 'openssl@1.1'])
E           Actual: check_output(['brew', 'install', 'openssl@3'])
```
Full log context:
https://github.com/kivy/python-for-android/actions/runs/12020321050/job/33508717552?pr=3086

Also do you know why some integration tests are failing?
https://github.com/kivy/python-for-android/actions/runs/12020321050/job/33508724907?pr=3086

--- a/tests/test_prerequisites.py
+++ b/tests/test_prerequisites.py
@@ -99,8 +99,8 @@ class TestOpenSSLPrerequisite(PrerequisiteSetUpBaseClass, unittest.TestCase):
         self.mandatory = dict(linux=False, darwin=True)
         self.installer_is_supported = dict(linux=False, darwin=True)
         self.prerequisite = OpenSSLPrerequisite()
-        self.expected_homebrew_formula_name = "openssl@1.1"
-        self.expected_homebrew_location_prefix = "/opt/homebrew/opt/openssl@1.1"
+        self.expected_homebrew_formula_name = "openssl@3"
+        self.expected_homebrew_location_prefix = "/opt/homebrew/opt/openssl@3"
 
     @mock.patch(
         "pythonforandroid.prerequisites.Prerequisite._darwin_get_brew_formula_location_prefix"

